src/parcsr_ls/par_lr_interp_device.c

```diff
@@ -26,6 +26,22 @@ __global__ void hypreCUDAKernel_compute_aff_afc_epe( HYPRE_Int nr_of_rows, HYPRE
 
 __global__ void hypreCUDAKernel_compute_dlam_dtmp( HYPRE_Int nr_of_rows, HYPRE_Int *AFF_diag_i, HYPRE_Int *AFF_diag_j, HYPRE_Complex *AFF_diag_data, HYPRE_Int *AFF_offd_i, HYPRE_Complex *AFF_offd_data, HYPRE_Complex *rsFC, HYPRE_Complex *dlam, HYPRE_Complex *dtmp );
 
+struct equal_pred : public thrust::unary_function<HYPRE_Int, bool>
```

Comment:
Can we use https://github.com/hypre-space/hypre/blob/f4336b818dc73ddc6cf0b3fb1fe4a978edf7227e/src/utilities/cuda_utils.h#L849?

Reply:
Yes, thanks.